pylint/lint/report_functions.py

```diff
@@ -27,23 +30,25 @@ def report_messages_stats(sect, stats, _):
         if not msg_id.startswith("I")
     )
     in_order.reverse()
-    lines = ("message id", "occurrences")
+    lines: Tuple[str, str] = ("message id", "occurrences")
```

Comment:
This should probably be `List[str]`. https://github.com/PyCQA/pylint/pull/4973#discussion_r708572347

Reply:
```suggestion
    lines: List[str] = ("message id", "occurrences")
```